### ISSUE TITLE
Video streams causing Home assistant to crash - Log entries attached

### ISSUE THREAD

**Issue by frank451a** [AlexxIT/WebRTC#716]
Hi

I hav erecently got into webrtc etc.  Having issues with one touchscreen android 7 client whereby it kills HA.  The error in the log is below.  Not sure what to do to resolve or fi its supported anymore.

There are 1000s of entries until HA gives up.  I cant update the android unit as its locked down  The android hardware is based on a iot3399-e similar to Firefly RK3399

2024-06-19 22:16:14.314 ERROR (MainThread) [frontend.js.latest.202406101] Uncaught error from Firefox 106.0 on Android 7.1.2
RangeError: source array is too long
onmse/this.onmessage.mse/this.ondata (/webrtc/video-rtc.js:463:25)
onopen/< (/webrtc/video-rtc.js:338:22)

**Comment by AlexxIT** [comment]
Are you using v3.5.2? I don't know why this line can generate this error:

https://github.com/AlexxIT/WebRTC/blob/ff5520880ce1d0553a8c6a5001067057f602c7ac/custom_components/webrtc/www/video-rtc.js#L463

**Comment by frank451a** [comment]
Yes 3.5.2 and go2rtc 1.9.4. Not sure if using that particular line. Thank you

**Comment by AlexxIT** [comment]
I can hardly replicate the same environment. On Desktop in Chrome it is not possible to get this kind of error.

**Comment by SkeLLLa** [comment]
@AlexxIT I could reproduce this error pretty constantly when using Firefox (dekstop/mobile) and hi-res stream (e.g. 2k or 4k). In my case it's connected to other issue: configured stream resolution doesn't work and stream is shown in native resolution. But I'm still trying to understand whether it connected to go2rtc or browser or HA card. So I think it's connected with some browser limits or so. 

So in order to reproduce: I use 2k stream h265, with up to 6Mb/s rate. 

And regarding chrome - it wasn't reproduced, because there go2rtc transcoding to lower resolution works :).

**Comment by AlexxIT** [comment]
@SkeLLLa thanks. Details for repeating an issue is always a nice thing.
